src/sentry/integrations/utils/codecov.py

```diff
@@ -29,13 +29,15 @@ def get_codecov_data(
                 url, params=params, headers={"Authorization": f"Bearer {codecov_token}"}
             )
             scope.set_tag("codecov.http_code", response.status_code)
```

Comment:
Could you please add one more tag in here?
`scope.set_tag("codecov.api_url", url + appended params)`

That way we know what exactly we called the URL with (bar the token).

Reply:
We have `codecov.attempted_url` in the caller method (`src/sentry/api/endpoints/project_stacktrace_link.py`) which should track the called URL.